package org.webrtc;

/* JADX INFO: loaded from: classes3.dex */
public interface BitrateAdjuster {
    int getAdjustedBitrateBps();

    double getAdjustedFramerateFps();

    void reportEncodedFrame(int i);

    void setTargets(int i, double d);
}
